src/azure-cli/azure/cli/command_modules/network/_actions.py

```diff
@@ -8,6 +8,7 @@
 import argparse
 from knack.util import CLIError
 from azure.cli.core.azclierror import UnrecognizedArgumentError
+from msrestazure.tools import is_valid_resource_id
```

Comment:
use `from azure.mgmt.core.tools import is_valid_resource_id` instead as we'll deprecate `msrestazure` in the future.